shell/edit/workload/index.vue

```diff
@@ -37,15 +37,25 @@ export default {
      */
     mapError(error) {
```

Comment:
Please add a note related to the different types of errors, since we do not use Typescript, nor we have tests in place.